tests/test_notebook.py

```diff
@@ -7,23 +7,24 @@
 
 
 this_dir = os.path.dirname(os.path.abspath(__file__))
-
+TESTS_NB_FILE = os.path.abspath(os.path.join(os.path.dirname(__file__), 'misc', 'memestra_nb_demo.ipynb'))
+import sys
+sys.path.insert(0, os.path.join(this_dir, 'misc'))
```

Comment:
I'm not really sure on how to fix this on the notebook tests though as it doesn't use a `file_path` argument. Any idea?

Reply:
Maybe https://nbconvert.readthedocs.io/en/latest/api/exporters.html#nbconvert.exporters.Exporter.from_notebook_node

Reply:
So, regarding this, I couldn't fix it. I'm not sure anymore if I tested it on mac or linux first. Just tested on Mac and can still reproduce my problem. @davidbrochart said all tests passed to him when he commented the `import sys` out.
So... I  should test on my Mac once I get home today.

Reply:
Tested on my linux, still crashing. Could you test locally @serge-sans-paille ? Commenting out the sys.path line and see if it works please?
Thanks.

Reply:
Works fine by me without the `sys.path` change

Reply:
Cool! Tx!